build/extensions.json

```diff
@@ -741,6 +741,19 @@
       },
       "url": "https://raw.githubusercontent.com/open-contracting-extensions/ocds_submissionTerms_extension/master/"
     },
+    {
+      "category": "tender",
+      "core": false,
+      "description": {
+        "en": "Adds a sustainability array and a hasSustainability field to the tender and lot levels to identify if this process is related to Sustainable Public Procurement (SPP)."
+      },
+      "documentation_url": "https://github.com/open-contracting-extensions/ocds_sustainability_extension",
```

Comment:
I guess we update it to the explorer in a later commit?

Reply:
Yes, I put that URL because a test fails if I leave the URL blank